charts/prometheus/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: prometheus
 appVersion: v2.45.0
-version: 22.6.7
+version: 22.7.0
```

Comment:
@kvanzuijlen 
Thanks for the contribution. Since you bump to minor version, can you write something that the user has to know for upgrading in README.md?

Reply:
Thanks for your suggestion and your review. README should be fine now :)